testutil/gstestdata.go

```diff
@@ -163,7 +165,12 @@ func NewGraphsyncTestingData(ctx context.Context, t *testing.T, host1Protocols [
 // SetupGraphsyncHost1 sets up a new, real graphsync instance on top of the first host
 func (gsData *GraphsyncTestingData) SetupGraphsyncHost1() graphsync.GraphExchange {
 	// setup graphsync
-	return gsimpl.New(gsData.Ctx, gsData.GsNet1, gsData.Loader1, gsData.Storer1)
+	if gsData.gs1Cancel != nil {
+		gsData.gs1Cancel()
+	}
```

Comment:
Is this in case the test data is reused in a test?

Reply:
yes